lib/kino/smart_cell.ex

```diff
@@ -391,4 +391,15 @@ defmodule Kino.SmartCell do
     |> Inspect.Algebra.format(90)
     |> IO.iodata_to_binary()
   end
+
+  def global_attr(cell, key) do
+    case Kino.Global.lookup("#{cell}-#{key}") do
```

Comment:
The key doesn't have to be a string. I would recommend storing a tuple as key:

    {Kino.RemoteCell, :node}
    {Kino.RemoteCell, :secret}

And then, you should also store the secret as a tuple. If it is a plain cookie, store it as `{value, nil}`. If it is a cookie secret, store it as `{nil, value}`. Then you can read it as:

    {global_cookie, global_cookie_secret} = SmartCell.global_attr(@global_key, :cookie, {nil, nil})

Where the third argument is the default value (which means you need to provide different names for the function to read and the function to write).

I am also not sure if we should call it global, because it is not shared across nodes. Maybe shared_attribute? So I would have:

    Kino.SmartCell.get_shared_attribute(cell, key, default \\ nil)
    Kino.SmartCell.put_shared_attribute(cell, key, value)

Also, if we want to make it public, we would need to add docs? Otherwise we add `@doc false`. :)

Reply:
I think it can stay private until we actually need it somewhere else. We don't expose the counter either, just the high-level variable generation.